pkg/apis/hostpathprovisioner/v1beta1/hostpathprovisioner_types.go

```diff
@@ -76,7 +100,7 @@ type HostPathProvisionerList struct {
 	Items           []HostPathProvisioner `json:"items"`
 }
 
-// PathConfig contains the information needed to build the path where the PVs will be created.
+// PathConfig contains the information needed to build the path where the PVs will be created. Deprecated
```

Comment:
Is it really deprecated in the same PR that introduces the new method?  Maybe we should wait.

Reply:
I want to support the old method as little as possible. Deprecation does not mean removal.